cmd/config/internal/commands/test/data/dataset1/mysql/Krmfile

```diff
@@ -0,0 +1,33 @@
+apiVersion: krm.dev/v1alpha1
```

Comment:
We should put test data under a folder called `testdata` so it will be ignored by the go tool: https://dave.cheney.net/2016/05/10/test-fixtures-in-go

Reply:
Done.